### PR TITLE
feature: allows working offline

### PR DESCRIPTION
**Pull request by OuiSouss**
# Feature 
- Use local source file

## Goal
Bypass downloading of source from github because, in offline mode, it is not possible to get sources from external.

*This feature is inspired of roles from cloudalchemy.*

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -10,14 +10,27 @@
   with_items:
     - /opt/cadvisor
 
-- name: cadvisor | download
-  become: true
-  get_url:
-    url: "{{ cadvisor_download }}"
-    checksum: "sha256:{{ cadvisor_checksum }}"
+- block:
```
Comment (sbesson) [inline tasks/main.yml]: What is the rationale for using a `block` as it only wraps one tasks (as opposed to attaching the `when` clause to `get_url` as for the `copy` task below?

**Review by sbesson — COMMENTED** [review]
Thanks for your contribution @OuiSouss. One inline question about the usage of `block`. Otherwise the change looks pretty straightforward and should add the flexibility of using a locally downloading binary version of `cadvisor` with the default behavior remaining unchanged.

In order for us to merge and release your changes, could you sign and return the OME CLA as per the instructions described [here](https://ome-contributing.readthedocs.io/en/latest/cla.html)?

File: tasks/main.yml (hunk 1 above)
Comment (OuiSouss) [reply tasks/main.yml]: You're right, there's no point in it being there. I think it's left over from another feature I think about. I'm going to remove it, right now. Thank for the comment

**Review by sbesson — APPROVED** [review]
Ready to merge once the CLA is sorted

**Comment by sbesson** [comment]
@OuiSouss `cadvisor 0.4.0` should now be published and available from https://galaxy.ansible.com/ome/cadvisor

**Comment by OuiSouss** [comment]
Thank you for your reactivity.

**Pull request merged**